addon/globalPlugins/nao.py

```diff
@@ -32,13 +34,25 @@
 pdfToPngToolPath = "\""+os.path.join (addonPath, "tools", "pdftopng.exe")+"\""
 pdfToImagePath = "" + os.path.join (addonPath, "images") + ""
 pdfToImageFileNamePath = pdfToImagePath + "\\img"
+ADDON_SUMMARY = addonHandler.getCodeAddon().manifest["summary"]
 
 class GlobalPlugin(globalPluginHandler.GlobalPlugin):
+	try:
+		scriptCategory = unicode(ADDON_SUMMARY)
```

Comment:
Since this add-on is only compatible with NVDA 2021.2 and above (thus always Python 3), this statement always raises the error; thus the try/except clause is not needed.